client/main.lua

```diff
@@ -142,36 +142,36 @@ RegisterNetEvent('ox_doorlock:setState', function(id, state, source, data)
 	if double then
 		-- Disabled a consistent methood of accelerating a locked door is found.
 		-- Some users reported doors getting stuck in the incorrect state.
-		-- while not door.auto and door.state == 1 and double[1].entity do
-		-- 	local doorOneHeading = double[1].heading
-		-- 	local doorOneCurrentHeading = math.floor(GetEntityHeading(double[1].entity) + 0.5)
+		while not door.auto and door.state == 1 and double[1].entity do
```

Comment:
It could be argued that you'd want to check for double[2].entity in this clause as well, but honestly could go either way.